nordicsemi/dfu/package.py

```diff
@@ -152,6 +155,9 @@ def __init__(self,
                                      filename=app_fw,
                                      init_packet_data=init_packet_vars)
 
+        if sd_req is not None:
+            init_packet_vars[PacketField.REQUIRED_SOFTDEVICES_ARRAY] = sd_req
```

Comment:
On line 148, `init_packet_vars[PacketField.REQUIRED_SOFTDEVICES_ARRAY]` might have already been set to `sd_id`. This would overwrite that.

If the same init packet field is being used for both `--sd-req` and `--sd-id`, then why do we need `--sd-id`? Couldn't we just use `--sd-req`?